apollo-router/src/configuration/expansion.rs

```diff
@@ -142,6 +144,18 @@ impl Expansion {
         }
     }
 
+    pub(crate) fn expand_env(&self, key: &str) -> Result<Option<String>, ConfigurationError> {
```

Comment:
I feel that the error message in here is targeted at config expansion.
The rhai expansion should have it's own error message.

Reply:
I want the `Rhai` functionality to look/smell the same, so I think this message is perfect. In the future we may add more configuration/expansion capabilities into `Rhai`, so deviation is probably not a good thing.